src/pages/OfferNew/Compare/mock.ts

```diff
@@ -70,7 +70,7 @@ export const otherCompanies: CompanyProperties[] = [
   },
   {
     name: 'Moderna Försäkringar Grundskydd',
-    id: 'Moderna Försäkringar',
+    id: 'Moderna',
```

Comment:
Ouff this feels dangerous. Has this casued any issues?

Reply:
Yeah, we couldn't get the trustpilot score for moderna.